pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -64,14 +75,15 @@
   libjpeg_turbo,
   giflib,
   protobuf-core,
+  hdf5,
   # Upstream by default includes cuda support since tensorflow 1.15. We could do
   # that in nix as well. It would make some things easier and less confusing, but
   # it would also make the default tensorflow package unfree. See
   # https://groups.google.com/a/tensorflow.org/forum/#!topic/developers/iRCt5m4qUz0
   config,
   cudaSupport ? config.cudaSupport,
   cudaPackages,
-  cudaCapabilities ? cudaPackages.cudaFlags.cudaCapabilities,
+  cudaCapabilities ? lib.optionals (config.cudaSupport) cudaPackages.cudaFlags.cudaCapabilities,
```

Comment:
I am not sure you should change this.

Reply:
also curious why this change is necessary

Reply:
> also curious why this change is necessary

Turns out it isn't. AFAIR, introducing a conditional was necessary to compile for non-CUDA on one of the earlier rebases during the work on this PR, when access to the CUDA packages worked differently (`cudaPackagesGoogle` etc.).